### PR TITLE
remove the {runnable} from our APIs where appropriate

### PR DESCRIPTION
**Pull request by anew**
Part of /v3 REST API cleanup

### CODE REVIEW

File: cdap-app-fabric/src/main/java/co/cask/cdap/app/ApplicationSpecification.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
 import java.util.Map;
 
 /**
- * Application Specification used in core code.
+ * Application Specification used in core code.P
```
Comment (sreevatsanraman) [inline cdap-app-fabric/src/main/java/co/cask/cdap/app/ApplicationSpecification.java]: Un-intended character 'P'

File: cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppFabricHttpHandler.java [hunk 2]
```diff
@@ -301,44 +301,44 @@ public void stopProgram(HttpRequest request, HttpResponder responder,
...
    */
   @GET
-  @Path("/apps/{app-id}/{runnable-type}/{runnable-id}/runs")
+  @Path("/apps/{app-id}/{program-type}/{program-id}/runs")
   public void runnableHistory(HttpRequest request, HttpResponder responder,
                               @PathParam("app-id") String appId,
-                              @PathParam("runnable-type") String runnableType,
-                              @PathParam("runnable-id") String runnableId,
+                              @PathParam("program-type") String runnableType,
+                              @PathParam("program-id") String runnableId,
```
Comment (sreevatsanraman) [inline cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppFabricHttpHandler.java]: We still have runnableType and runnableId in the variable.

**Comment by sreevatsanraman** [comment]
There are a few places (variable names) where we use runnableType and runnableId. Can we fix those as well? LGTM other than those comments.

**Comment by awholegunch** [comment]
We also have numerous references in the docs. Did you want to fix this as part of this or another PR?

File: cdap-app-fabric/src/main/java/co/cask/cdap/app/ApplicationSpecification.java (hunk 1 above)
Comment (anew) [inline cdap-app-fabric/src/main/java/co/cask/cdap/app/ApplicationSpecification.java]: Oops - and unnoticed. Will fix

File: cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppFabricHttpHandler.java (hunk 2 above)
Comment (anew) [inline cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppFabricHttpHandler.java]: will fix.

**Comment by anew** [comment]
@awholegunch where in the docs? Can you send me a reference?

**Comment by anew** [comment]
@sreevatsanraman I fixed the remaining ones - except where a method takes a programId and a runnable id (runnable being a flowlet or a service runnable, which will get fixed in a separate PR I hope).

**Comment by anew** [comment]
@awholegunch I searched through the docs, I could only find it in the context where it means flowlet or handler of a service. For these cases we are keeping this notation for now. But we are removing the "runnable" notation from services, and then we should make the doc change, too.

**Pull request merged**